### PR TITLE
Add missing ns2.0 InteropServices apis to model.xml

### PR DESCRIPTION
**Pull request by alexperovich**

### CODE REVIEW

File: src/mscorlib/model.xml [hunk 1]
```diff
@@ -6867,6 +6882,15 @@
     <Type Name="System.Runtime.InteropServices.PreserveSigAttribute">
       <Member Name="#ctor" />
     </Type>
+    <Type Name="System.Runtime.InteropServices.RuntimeEnvironment">
```
Comment (jkotas) [inline src/mscorlib/model.xml]: This should not be added. @ramarag build a standalone implementation in corefx.

File: src/mscorlib/model.xml [hunk 2]
```diff
@@ -6703,6 +6712,8 @@
       <Member Name="AddRef(System.IntPtr)" />
       <Member Name="AllocHGlobal(System.Int32)" />
       <Member Name="AllocCoTaskMem(System.Int32)" />
+      <Member Name="BindToMoniker(System.String)" />
+      <Member Name="ChangeWrapperHandleStrength(System.Object,System.Boolean)" />
```
Comment (jkotas) [inline src/mscorlib/model.xml]: This won't build without matching changes in the VM

**Comment by jkotas** [comment]
cc @yizhang82

File: src/mscorlib/model.xml (hunk 1 above)
Comment (jkotas) [inline src/mscorlib/model.xml]: See https://github.com/dotnet/corefx/pull/12574

Comment (alexperovich) [inline src/mscorlib/model.xml]: I will remove it.

**Comment by alexperovich** [comment]
@jkotas PTAL

File: src/mscorlib/model.xml (hunk 2 above)
Comment (yizhang82) [inline src/mscorlib/model.xml]: @weshaggard Do you know why are these two methods included? They are not in S.R.InteropServices and not in the master API list I've seen. These two APIs are not interesting.

**Comment by jkotas** [comment]
@dotnet-bot test this please

**Comment by yizhang82** [comment]
cc @shrah FYI

File: src/mscorlib/model.xml (hunk 2 above)
Comment (jkotas) [inline src/mscorlib/model.xml]: They are in https://raw.githubusercontent.com/dotnet/standard/master/netstandard/ref/mscorlib.cs

**Comment by alexperovich** [comment]
@dotnet-bot test this please

**Comment by alexperovich** [comment]
@dotnet-bot test this please

File: src/mscorlib/model.xml (hunk 2 above)
Comment (weshaggard) [inline src/mscorlib/model.xml]: These were included because they were in the intersection of .NET 4.6.1 and Xamarin.ios

**Comment by alexperovich** [comment]
@dotnet-bot test Linux ARM Emulator Cross Debug Build please (*** stack smashing detected ***)

**Comment by alexperovich** [comment]
@dotnet-bot test Linux ARM Emulator Cross Debug Build please

**Pull request merged**